### ISSUE TITLE
Wokflow 'Copy' reporting succeeded on large clinvar file but the file was only partially copied

### ISSUE THREAD

**Issue by toneillbroad** [clingen-data-model/clinvar-ingest#93]
Testing the workflow against a real clinvar variation release file fails with the following message:

```
RuntimeError: "Step parse failed for ClinVarVariationRelease_2024-0107.xml.gz Failed to parse gs://clinvar-ingest/executions/2024_01_07_2024_02_02T153731743700/clinvar_xml/ClinVarVariationRelease_2024-0107.xml.gz and write to gs://clinvar-ingest/main-release-outputs: Not a gzipped file (b'\u003c!')"
in step "step_failed", routine "main", line: 68
"Step parse failed for ClinVarVariationRelease_2024-0107.xml.gz Failed to parse gs://clinvar-ingest/executions/2024_01_07_2024_02_02T153731743700/clinvar_xml/ClinVarVariationRelease_2024-0107.xml.gz and write to gs://clinvar-ingest/main-release-outputs: Not a gzipped file (b'<!')"
```

Even though the error message is complaining about the "parse" step not working, it is actually the "copy" step that is failing. Although it does seem to work for the small file we tested with initially. 

The "step_status" call  is returning that the copy was successful even though it was not successful.

**Comment by toneillbroad** [comment]
Issue was due to the read call returning 0 bytes - which indicated to the process that it was finished writing.

**Comment by theferrit32** [comment]
Closed in #91
